scripts/smart-dump.ts

```diff
@@ -34,37 +38,51 @@ program.command('dump [recipe] [env]').action(async (recipe, env) => {
     recipe = './smart-dump/defaultRecipe.js';
   }
 
+  const tempDumpDir = fs.mkdtempSync(path.join(os.tmpdir(), 'oc-export-'));
```

Comment:
Not sure why the dir name ends with `-`, you maybe wanted to add a unique ID there?

Reply:
`mkdtempSync` generates a unique temp directory with a given prefix, so this results in something like `/tmp/os-export-c122Fuytasd`